### PR TITLE
Add multicut

### PR DESCRIPTION
**Pull request by k-dominik**
added executor for pre-trained multicut projects

### CODE REVIEW

File: src/main/java/org/ilastik/ilastik4ij/ui/IlastikMulticutCommand.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+        try {
+            runClassification();
+        } catch (IOException e) {
+            logService.error("Multicut command failed", e);
+            throw new RuntimeException(e);
+        }
+    }
+
+    private void runClassification() throws IOException {
```
Comment (emilmelnikov) [inline src/main/java/org/ilastik/ilastik4ij/ui/IlastikMulticutCommand.java:65]:
```suggestion
    private void runMulticut() throws IOException {
```

File: src/main/java/org/ilastik/ilastik4ij/ui/IlastikMulticutCommand.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+     * Run method that calls ilastik
+     */
+    @Override
+    public void run() {
+
+        if (ilastikOptions == null)
+            ilastikOptions = optionsService.getOptions(IlastikOptions.class);
+
+        try {
+            runClassification();
```
Comment (emilmelnikov) [inline src/main/java/org/ilastik/ilastik4ij/ui/IlastikMulticutCommand.java:58]:
```suggestion
            runMulticut();
```

File: src/main/java/org/ilastik/ilastik4ij/executors/ObjectClassification.java [hunk 3]
```diff
@@ -29,6 +29,7 @@ public ObjectClassification(File executableFilePath, File projectFileName, LogSe
     protected List<String> buildCommandLine(Map<String, String> tempFiles, PixelPredictionType secondInputType) {
 
         List<String> commandLine = getBaseCommand();
+        commandLine.add("--raw_data=" + tempFiles.get(rawInputTempFile));
```
Comment (emilmelnikov) [inline src/main/java/org/ilastik/ilastik4ij/executors/ObjectClassification.java:32]: Leftover?

**Review by emilmelnikov — APPROVED** [review]

File: src/main/java/org/ilastik/ilastik4ij/executors/ObjectClassification.java (hunk 3 above)
Comment (k-dominik) [reply src/main/java/org/ilastik/ilastik4ij/executors/ObjectClassification.java:32]: hm, nah, I even made it an extra commit. I don't think Object Classification was working properly previously...

Comment (emilmelnikov) [reply src/main/java/org/ilastik/ilastik4ij/executors/ObjectClassification.java:32]: After looking at the surrounding code, this makes sense.

Maybe also explicitly mention this fix in the merge commit message?

**Pull request merged**